src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -162,8 +162,29 @@
       <!-- In .NET 6 the browser-wasm runtime pack started using the Mono naming pattern -->
       <Net60RuntimePackRids Remove="browser-wasm" />
 
-      <NetCoreAppHostRids Include="@(Net60AppHostRids)" />
-      <NetCoreRuntimePackRids Include="@(Net60RuntimePackRids)" />
+      <Net70AppHostRids Include="
+          @(Net60AppHostRids);
+          linux-bionic-arm;
+          linux-bionic-arm64;
+          linux-bionic-x64;
+          linux-bionic-x86;
+          "/>
+
+      <Net70RuntimePackRids Include="
+          @(Net60RuntimePackRids);
+          linux-bionic-arm;
+          linux-bionic-arm64;
+          linux-bionic-x64;
+          linux-bionic-x86;
+          " />
+
+      <NetCoreAppHostRids Include="
```

Comment:
Out of curiosity, is it advantageous or somewhat necessary to keep the the naming `NetCoreAppHostRids`/`NetCoreRuntimePackRids`? If we are just going to bump https://github.com/dotnet/installer/blob/fe8ff26c2f24d1a92cfdf04a945dc820a818e7db/src/redist/targets/GenerateBundledVersions.targets#L357-L374 to point at `Net70AppHostRids`/`Net70RuntimePackRids` were a newer version to come out, it feels like a superfluous ItemGroup. Perhaps we can do away with `NetCoreAppHostRids`/`NetCoreRuntimePackRids` and directly make them `Net70AppHostRids`/`Net70RuntimePackRids`?